lib/taskana-history-plugin/src/main/java/pro/taskana/history/plugin/HistoryEvent.java

```diff
@@ -0,0 +1,74 @@
+package pro.taskana.history.plugin;
+
+import java.time.Instant;
+
+/**
+ * Interface for HistoryEvent. This model-object contains information about HistoryEvents structure.
+ */
+public interface HistoryEvent {
+
+    /**
+     * Gets the id of the historyEvent.
+     *
+     * @return eventId
+     */
+    String getId();
+
+    /**
+     * Gets the business process id of the historyEvent.
+     *
+     * @return businessProcessId
+     */
+    String getBusinessProcessId();
+
+    /**
+     * Gets the parent business process id of the historyEvent.
+     *
+     * @return parentBusinessProcessId
+     */
+    String getParentBusinessProcessId();
+
+    /**
+     * Gets the task id of the historyEvent.
+     *
+     * @return taskId
+     */
+    String getTaskId();
+
+    /**
+     * Gets the event type of the historyEvent.
+     *
+     * @return eventType
+     */
+    String getEventType();
+
+    void setEventType(String historyEvent);
```

Comment:
JavaDoc missing

Reply:
Why is the event type a string and not an enum? This can lead to inconsistency

Reply:
We have not defined which operations are we going to have, I prefer keep it this way until we have a better description of the operations